js/menu.js

```diff
@@ -0,0 +1,14 @@
+const btn = document.querySelector('.mb-menu');
```

Comment:
- [ ] I tried entering an email with an uppercase and it accepted it, let's make sure we add validation for the email to only be lowercase. and also display error for the user if it occurred for a better user experience

Reply:
Hey, thank you for the feedback, please check my update now if there is anything else to fix